### DISCUSSION TITLE
Any way to add a visual description to a workflow?

### DISCUSSION THREAD

**Discussion by xahon** [argoproj/argo-workflows#8628] [Q&A] [answered]
I've seen an `annotations:` field in this workflow https://github.com/argoproj/argo-workflows/blob/master/examples/coinflip.yaml

I tried it in practice and didn't notice any changes. I'd like to make a generated description that would be available to developers to understand what a specific workflow actually did. The only way it is useful is to look up for a manifest of the workflow and find this field in it

I didn't find any mentions of `annotations:` in argo workflow documentation

**Comment by terrytangyuan** [comment, marked as answer]
It’s currently not used in the UI. There’s an issue to track that feature request: https://github.com/argoproj/argo-workflows/issues/6529
